### DISCUSSION TITLE
TPM verse TMM?

### DISCUSSION THREAD

**Discussion by srwis** [omnideconv/immunedeconv#70] [Q&A] [answered]
Hi!
I have RNA-Seq data that has already been TMM normalized via EdgeR and is CPM units. How wrong would it be to used that instead of renormalizing as TPM? I don't have easy access to the sequence lengths. Seems like it may not impact the interpretation of results for methods like MCP-Counter but is it totally wrong for something like quantiSeq?
Thanks!
Sadie

**Comment by grst** [comment, marked as answer]
Hi Sadie, 

The deconvolution-based methods (CIBERSORT, Timer, quanTIseq, EPIC) definitely require TPM values. I would expect the marker-gene based methods (MCP-counter and xCell) to work reasonably well with different normalization approaches, but I never tried with CPM values and would still be cautious. 

Best,
Gregor
